src/odemis/util/spot.py

```diff
@@ -147,70 +157,75 @@ def FindCenterCoordinates(image, smoothing=True):
     Returns
     -------
     pos : tuple
-        Position of the radial symmetry center in px from the center of the
-        image.
+        Position of the radial symmetry center `(dj, di)` in pixels relative to
```

Comment:
In Odemis, the convention is to report positions as (x,y). I don't see the point of having this function behave differently.
Admittedly, we have some exceptions for values which could be directly used as index in a numpy array, but that's it. Here, the position is relative to the center *and* floats, so it's never going to be directly used an array index. So stick to x,y.

Reply:
The problem is that `FindCenterCoordinates` returns coordinates as `(x, -y)`. That's confusing as hell.
Since this function doesn't care about the physical size I thought it would make sense to use the pixel coordinates (following the discussions we had in September last year), and then convert to physical coordinates using `to_physical_space` when appropriate.